mbed_targets/_internal/target_attribute_hierarchy_parsers/accumulating_attribute_parser.py

```diff
@@ -13,18 +13,31 @@
 from typing import Dict, List, Any, Deque
 
 ACCUMULATING_ATTRIBUTES = ["extra_labels", "macros", "device_has", "features", "components"]
```

Comment:
Should this be a `tuple` rather than a `list` if we're saying it's a constant? Mutable constants make my head hurt :)

Reply:
45ed7ec